glide.lock

```diff
@@ -126,11 +240,87 @@ imports:
   - pkg/provenance
   - pkg/repo
   - pkg/resolver
+  - pkg/storage/driver
   - pkg/strvals
   - pkg/sympath
   - pkg/tlsutil
   - pkg/urlutil
   - pkg/version
+- name: k8s.io/kubernetes
```

Comment:
Dislike the huge additional dependency here, but `glide update` added it anyway.

Reply:
Urks. How did the helm people managed to bring that back in... :(

Reply:
Ah man. It seems the only reason k8s.io/kubernetes is pulled in because of `fake.go` in k8s.io/helm/pkg/helm. And only because they use 3 errors from the `k8s.io/helm/pkg/storage` in there. This is so unnecessary...

Reply:
This is clearly an accident. We should try to fix this upstream. We have had this before. https://github.com/helm/helm/pull/2685

Reply:
So just by replacing `storage.ErrReleaseNotFound(rlsName)` with `fmt.Errorf("release: %q not found", rlsName)` three times in `fake.go` the kubernetes dependency goes away from `k8s.io/pkg/helm`

Reply:
I'm opening a PR upstream